packages/server/src/fhir/lookups/lookuptable.ts

```diff
@@ -140,17 +140,23 @@ export abstract class LookupTable<T> {
    * @param client - The database client.
    * @param resourceType - The resource type.
    * @param values - The values to insert.
+   * @param ignoreConflict - Whether to ignore conflicts when inserting lookup values.
    */
   protected async insertValuesForResource(
     client: Pool | PoolClient,
     resourceType: ResourceType,
-    values: Record<string, any>[]
+    values: Record<string, any>[],
+    ignoreConflict?: boolean
   ): Promise<void> {
     if (values.length === 0) {
       return;
     }
     const tableName = this.getTableName(resourceType);
-    await new InsertQuery(tableName, values).execute(client);
+    const insert = new InsertQuery(tableName, values);
+    if (ignoreConflict) {
```

Comment:
What is the case for conflicts?

For better or worse, we currently delete all values before insert, so there should never be a conflict (at least that's my mental model).

Reply:
This was necessary to make a test pass where an `AuditEvent` had the same `entity` represented multiple times.  This could happen if, for example, a given Practitioner was involved in an activity in multiple roles.  Since the primary key for the new lookup table covers all columns, ignoring on conflict has the same effect as overwriting the row